internal/mount/mount_fuse.go

```diff
@@ -40,13 +59,14 @@ func Directory(ctx context.Context, entry fs.Directory, mountPoint string) (Cont
 
 	rootNode := fusemount.NewDirectoryNode(entry)
 
-	fuseConnection, err := fuse.Mount(
-		mountPoint,
+	options := append(
+		mountOptions.toFuseMountOptions(),
 		fuse.ReadOnly(),
 		fuse.FSName("kopia"),
 		fuse.Subtype("kopia"),
-		fuse.VolumeName("Kopia"),
-	)
+		fuse.VolumeName("Kopia"))
```

Comment:
Aren't all these options included in `toFuseMountOptions()`? Should this be simplified to simply use the options returned by `mountOptions.toFuseMountOptions()` ?